tests/tools/oecertdump/README.md

```diff
@@ -0,0 +1,13 @@
+oecertdump
+=====
+
+oecertdump is a utility that generates and validates reports and certificates.
+Generation and validation logs are saved to a log file specified by the --out option.
+Default log filename is "oecertdump_out.log".
+
+Usage: oecertdump ENCLAVE_PATH Options
+
+where Options are:
+    --out FILENAME        : specify output filename.
+
+Example: host/oecertdump enc/oecertdump_enc --out myoutput.log
```

Comment:
What's the standard output will look like if the test succeeded or failed?

Reply:
Added return value description